tests/test_provider.py

```diff
@@ -17,6 +17,8 @@
         ("optimism", "goerli"),
         ("polygon", "mainnet"),
         ("polygon", "mumbai"),
+        ["linea", "mainnet"],
```

Comment:
```suggestion
        ("linea", "mainnet"),
```

Reply:
sorry, my mistake, have changed to use ();